.dockerignore

```diff
@@ -0,0 +1,23 @@
+#Markdown files
+*.md
+
+#Docker files
+docker-compose.yml
+
+#Git files
+.git
```

Comment:
Please remove .git directory too, it's needed for build

Reply:
Done!